ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/DataStorageConfiguration.java

```diff
@@ -47,6 +47,8 @@ interface Unstable {
     long DEFAULT_BONSAI_TRIE_LOG_RETENTION_THRESHOLD = 512L;
     long MINIMUM_BONSAI_TRIE_LOG_RETENTION_THRESHOLD = DEFAULT_BONSAI_TRIE_LOG_RETENTION_THRESHOLD;
     int DEFAULT_BONSAI_TRIE_LOG_PRUNING_LIMIT = 30_000;
+    boolean DEFAULT_BONSAI_CODE_USING_CODE_HASH_ENABLED = false;
+    boolean DEFAULT_BONSAI_CODE_USING_CODE_HASH_DELETION_ENABLED = true;
```

Comment:
shouldn't the default be false here too?

Reply:
I want the default to be true as generally, we should delete code from storage as part of a self-destruct. The special case is for tracing which most people won't use where could in the future use the code along with the trie logs to do a trace without needing to roll the world state.

Reply:
My main concern is, that this is a change in the current behavior that is not documented anywhere because this PR isn't adding any changelog since this is introduced under an experimental flag, is my understanding correct?